accumulo/src/main/java/com/yahoo/ycsb/db/AccumuloClient.java

```diff
@@ -276,11 +277,14 @@ public Status scan(String t, String startkey, int recordcount,
         }
         result.add(currentHM);
       }
+
+      Preconditions.checkNotNull(currentHM, "Unexpected scanner behaviour - currentHM is null");
```

Comment:
To use this, we'll have to add a dependency on guava to the accumulo binding (because Accumulo might stop including it). Can we switch to a vanilla assert instead?